engine/crates/engine-v2/schema/src/walkers/mod.rs

```diff
@@ -41,7 +41,10 @@ where
         self.id
     }
 
-    pub fn walk<OtherId>(self, id: OtherId) -> SchemaWalker<'a, OtherId> {
+    pub fn walk<OtherId>(self, id: OtherId) -> SchemaWalker<'a, OtherId>
+    where
+        OtherId: Copy,
```

Comment:
I don't understand why this bound, if it's not copy it won't change the method body, right?

Reply:
hum a good point, not sure why I've added it. Will remove it